ui/Screen/SendFunds.svelte

```diff
@@ -0,0 +1,258 @@
+<script>
+  import { getContext } from "svelte";
+  import {
+    TransferState,
+    payerStore,
+    recipientStore,
+    amountStore,
+  } from "../src/transfer.ts";
+
+  import { Dropdown, ModalLayout, Rad } from "../DesignSystem/Component";
+  import Row from "../DesignSystem/Component/Transaction/Row.svelte";
+  import {
+    Avatar,
+    Button,
+    Icon,
+    Input,
+    Text,
+    Title,
+  } from "../DesignSystem/Primitive";
+
+  const { identity } = getContext("session");
+  const { orgs } = getContext("session");
+
+  // pre-populate inputs with store values
+  let amount = $amountStore,
+    recipient = $recipientStore,
+    payer = $payerStore;
+
+  const dropdownOptions = [];
+  dropdownOptions.push({
+    variant: "avatar",
+    value: identity.accountId,
+    avatarProps: {
+      variant: "circle",
+      title: identity.metadata.handle,
+      avatarFallback: identity.avatarFallback,
+      imageUrl: identity.imageUrl,
+    },
+  });
+  for (let i = 0; i < orgs.length; i++) {
+    dropdownOptions.push({
+      variant: "avatar",
+      value: orgs[i].accountId,
+      avatarProps: {
+        variant: "square",
+        title: orgs[i].id,
+        avatarFallback: orgs[i].avatarFallback,
+        imageUrl: null,
+      },
+    });
+  }
+
+  let state = TransferState.Preparation;
+
+  const nextStep = () => {
+    state = TransferState.Confirmation;
+  };
+
+  const previousStep = () => {
```

Comment:
```suggestion
  const cancel = () => {
```

Reply:
`back` is maybe fine too

Reply:
I like `previousStep` better here since it makes it explicit that we are not cancelling the whole transfer, we are just going one step back:

``` rust
const previousStep = () => {
    state = TransferState.Preparation;
};
```

Reply:
Makes sense to name them more explicitly: https://github.com/radicle-dev/radicle-upstream/pull/712/commits/6b4d7fb0eb344641d5bd1f575446057ce0fc6ea4